app/_includes/md/gateway/setup.md

```diff
@@ -26,6 +26,12 @@ First, you must configure {{site.base_gateway}} using the `kong.conf` configurat
 For more information on how to configure {{site.base_gateway}} to connect to your database, see the Datastore section of the
 [Configuration property Reference](/gateway/{{ include.kong_version }}/reference/configuration/#datastore-section).
 
+<div class="alert alert-ee red">
+    <strong>Deprecation:</strong> 
+    Support for Cassandra is deprecated with <code>Kong Gateway</code> 2.7 and fully removed with 4.0. It is strongly discouraged to use Cassandra as your datastore as it will have limited support throughout the deprecation period. For more
+    information, see <a href="/gateway-oss/{{ link_to_cassandra_deprecation_overview }}">this blog post</a>.
+</div>
+
```

Comment:
```suggestion
{:.warning .no-icon}
> **Deprecation warning:** Support for Cassandra is deprecated with 
{{site.base_gateway}} 2.7 and planned to be fully removed with 4.0. It is strongly 
discouraged to use Cassandra as your datastore, as it will have limited support 
throughout the deprecation period. For more information, see 
[this blog post](add-some-link-here).
```

Using the newer (cleaner) formatting for alerts.

Reply:
Since this snippet gets reused a lot, I recommend moving it into an include. If you want to do that yourself, we have docs for it here: https://docs.konghq.com/contributing/includes/ 

Or I can make that change, if you prefer.

Reply:
I'll take care of that, thanks!